foot-in-the-door/pages/index.js

```diff
@@ -31,15 +31,10 @@ const events = [
 
 const videos = [
   {
-    vimeoId: 471541996,
-    caption: 'Foot in the Door 5 Artist Video: Ilene Krug Mojsilov',
+    vimeoId: 480895859,
+    caption: 'Foot in the Door 5 Artist: Eyenga Bokamba',
   },
-  {
-    vimeoId: 475231372,
-    caption: 'Foot in the Door 5 Artist: Alonzo Pantoja',
-  },
-]
-
```

Comment:
Even with all the validation in place, this in invalid javascript that breaks the build? The array needs to be closed with a `]`